packages/application-generic/src/services/workers/worker-base.service.ts

```diff
@@ -18,7 +19,7 @@ export type WorkerProcessor =
 
 export { WorkerOptions };
 
-export class WorkerBaseService {
+export class WorkerBaseService implements INovuWorker {
```

Comment:
This class is the base for all of the workers meaning it should implement the worker interface more than that he is responsible for implementing it.
so I see no reason for the duplication in all of the worker child classes.